mpa/modules/ov/models/__init__.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (C) 2022 Intel Corporation
```

Comment:
Why are we trying to import (mmcls/mmdet/mmseg)?
(Currently, many warning logs appear when importing `mmseg`, which is not public.)

Reply:
It was to register each `MMOV` model to a library specific registry.
Could you share warning logs?

Reply:
Sorry for the confusion. I'm not sure if here is wrong. It seems to be importing `nncf.torch` from somewhere.. (Even though nncf isn't working, it looks like it's importing somewhere.)

and, Is this import work to check if import is possible in advance?
Shouldn't this be inside the folders in `mmcls`, `mmdet`, `mmseg` respectively?
I don't know if it's appropriate to import `mmdet` and `mmseg` once to use classification (Ex).

Reply:
![image](https://user-images.githubusercontent.com/38045080/207227428-910e512c-a054-4ed9-8029-c46e7571440a.png)
I checked this again. I confirmed that unnecessary importing (& logs) do not come out by deleting the test import.
Could you please remove this?

Reply:
I can confirm that using public `mmseg` library, the warning messages is not appeared.
![image](https://user-images.githubusercontent.com/11530592/207232630-437c15f0-62ac-4a20-80f7-ecde3b6408cf.png)
Do you want me to comment them out for the moment?

Reply:
Commented them out in  https://github.com/openvinotoolkit/model_preparation_algorithm/pull/95/commits/9d4dea34e7d1b146f339a5aefe9d46757238a18a